.github/workflows/issue-labels.yml

```diff
@@ -0,0 +1,21 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+name: Label issues
+on:
+  issues:
+    types: [opened]
+
+jobs:
+  label_issues:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    steps:
+      - name: Label issues
+        uses: mheap/github-action-required-labels@v2
+        with:
+          add-labels: "needs-triage"
+          ignore-if-labeled: true
```

Comment:
If I label an issue with `area:share` during issue creation, will the `ignore-if-labeled` ignore the `needs:triage`?

Reply:
it will not add the `needs:triage` label if a label is already added. Only if people open issues on node repo without adding any labels AFAIK